scss/components/_login.scss

```diff
@@ -0,0 +1,56 @@
+.login {
+    @extend .mh-100;
+
+    &__container {
+        @extend .container;
+        @extend .mx-auto;
+        @extend .m-5;
```

Comment:
Does it actually work to extend .m-5? Didn't work for me locally

Reply:
I think those classes are on another file, I'll try to find which one if any.